ape_alchemy/provider.py

```diff
@@ -1,8 +1,10 @@
-import os
+import os, time
+import numpy.random as random
```

Comment:
Is there any way we can use `random` instead? It is a built-in module to python. `numpy` is probably an indirect dependency of Ape and this plugin, however it is still protocol to include it in the `setup.py`... Thus, I think if we could use the built-in random module, it would be better. It has the same method that you use here `randint`

```suggestion
import random
```